### ISSUE TITLE
How to restore profile from just ~/.ssb/secret?

### ISSUE THREAD

**Issue by stephenreid321** [ssbc/patchwork#1305]
I don't have gossip.json or conn.json - can anyone help?

**Comment by christianbundy** [comment]
Here's a zip with my `conn.json`, my guess is that we probably have some servers in common. Can you unzip it and plop that file into `~/.ssb/conn.json` and then start Patchwork? (Please don't modify `conn.json` while Patchwork is running.)

[conn.json.zip](https://github.com/ssbc/patchwork/files/4962108/conn.json.zip)

**Comment by stephenreid321** [comment]
Hmm, not a lot is happening. I just get a long list of hashes under 'Possible connections' on the left hand side.

**Comment by christianbundy** [comment]
@stephenreid321 Could you try connecting to some of those? My server is `@samhAxb0x` but I'm not 100% sure that it'll have your messages.

**Comment by agentlewis** [comment]
I was in this same situation and pulled down the zip and unpacked it to `~/.ssb/conn.json` next to my update `secret`.

When I opened Patchwork it loaded a bunch of connections - I just clicked connect on a bunch of them. There to search or filter for a specific one.

Nothing really happened and I just left it in the background. Then I heard a "wwaaaaaahhhh" - the tell tell sound of the fan spinning up as the db gets indexed.

I can see some of my posts from 2-3 years ago - so pretty sure it will have worked at this point.

Thanks for the fix.

Once its done I'll make a backup of the `conn.json`
